torch/csrc/jit/frontend/ir_emitter.cpp

```diff
@@ -523,6 +523,18 @@ struct Environment {
     return getSugaredVar(ident, range)->asValue(range, method);
   }
 
+  bool removeVar(const std::string& ident) {
+    bool removed = false;
+
+    for (auto runner = this; runner; runner = runner->next.get()) {
+      auto a = runner->value_table.erase(ident);
+      auto b = runner->type_table.erase(ident);
+      removed = (a + b) > 0;
```

Comment:
nit maybe `removed = a || b`;  with the `auto` it took me a sec to parse why we were adding them together.